### PR TITLE
Devx 2433 use case new styles

### PR DESCRIPTION
**Pull request by fabianrbz**
## Description

Related [ticket](https://nexmoinc.atlassian.net/browse/DEVX-2433)
Updates the styles for `/use-cases`

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: app/views/tutorial/list.html.erb [hunk 1]
```diff
@@ -1,14 +1,12 @@
 <h1>Tutorials</h1>
 
-<div class="Nxd-use-cases--filters">
-  <strong>Filter: &nbsp;&nbsp;</strong>
+<p>Get started with Nexmo with tutorials that will walk you through building a variety of practical applications</p>
```
Comment (benhayehudi) [inline app/views/tutorial/list.html.erb:3]: Should this be named to `Vonage` per the other rebrand PR?

Comment (fabianrbz) [reply app/views/tutorial/list.html.erb:3]: I think that's being done in the other PR, this one focuses on changing the page layout/design

**Comment by mheap** [comment]
A couple of things:

* I expected that clicking anywhere on the card on https://nexmo-develo-devx-2433--m58jpu.herokuapp.com/tutorials would take me to the tutorial
* Clicking on a language on a use case filters to that language, but clicking a language on a tutorial takes you to that implementation

**Comment by fabianrbz** [comment]
@mheap making the card clickable is not that easy, because we have nested elements that can be clickable, we could instead remove the `cursor: pointer` from it

**Comment by mheap** [comment]
Yes please, remove `cursor: pointer`

On tutorials if I filter to Kotlin only, then click a tutorial, I'd expect to see the Kotlin implementation rather than the JS one

**Comment by fabianrbz** [comment]
Included in https://github.com/Nexmo/nexmo-developer/pull/3036

**Pull request closed without merging**